account_analytic_distribution_manual/migrations/16.0.2.3.0/post-migration.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_analytic_line AS aal
```

Comment:
This should be also on post_init_hook

Reply:
Do you mean that in post_init_hook the `account.analytic.line` that have `account_analytic_tag` should be linked with the corresponding `manual_distribution_id`? That will only work if there is already account analytic distribution manual created and they have exactly the **same** name as the analytic tag (because they are not linked in any way).